converters/fromZigbee.js

```diff
@@ -40,6 +40,26 @@ const toPercentage = (value, min, max) => {
     return (normalised * 100).toFixed(2);
 };
 
+const toPercentageCR2032 = (voltage) => {
+    let Percentage = null;
```

Comment:
Could you change this to percentage? (its a naming convention to start local variables with a lowercase)